client/src/pages/Search.js

```diff
@@ -519,12 +529,13 @@ class Search extends Page {
 
 	@autobind
 	exportSearchResults() {
-		this._dataFetcher(this.props, this._exportSearchResultsCallback, 0, 0)
+		this._dataFetcher(this.props, this._exportSearchResultsCallback, 0)
```

Comment:
Why was this change from `candidate-2.1` reverted?

Reply:
I was too eager to introduce the redux paginated pageNum everywhere.
The export function obviously does not need paginated results.